generators/client/templates/vue/package.json

```diff
@@ -42,14 +42,14 @@
     "file-loader": "6.2.0",
     "fork-ts-checker-webpack-plugin": "6.2.10",
     "html-webpack-plugin": "5.3.1",
-    "jest": "26.6.3",
+    "jest": "27.0.4",
     "jest-junit": "12.1.0",
     "jest-serializer-vue": "2.0.2",
     "jest-sonar-reporter": "2.0.0",
     "jest-vue-preprocessor": "1.7.1",
     "merge-jsons-webpack-plugin": "1.0.21",
     "mini-css-extract-plugin": "1.6.0",
-    "node-notifier": "10.0.0",
+    "node-notifier": "9.0.1",
```

Comment:
What is the reason for the downgrade, any compatibility issues?

Reply:
Peer dependency conflict.

Reply:
Next jest releases allows version 10.

https://github.com/facebook/jest/blob/d1882f2e6033186bd310240add41ffe50c2a9259/packages/jest-core/package.json#L52

Reply:
Thanks for the explanation! Are we good to merge with this PR or do you need to add something?